### PR TITLE
[0.5.0] doc: review readmes and api 

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1584

* **Other information**:

I will create a PR for development at the end.

### CODE REVIEW

File: lib/README.md [hunk 1]
```diff
@@ -92,20 +114,40 @@ JWT=$(curl -s -X POST http://localhost:8000/auth \
...
+
+**Note:** If you need to the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
+
+How to remove a node from Dojot:
+
+```sh
+curl -X DELETE -H "Authorization: Bearer ${JWT}" http://localhost:8000//flows/v1/node/<node name> -H 'content-type: application/json'
+```
+
+**ATTENTION**: The `id` to add the node via API (when request `/flows/v1/node`) must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`. And within the html called in the `getNodeRepresentationPath` method also in the class that extends `dojot.DataHandlerBase`, the references inside the html `data-template-name=`, `data-help-name=`, `registerType(..` ,  and `RED._("...` must have this same `id`/`name`.
```
Comment (eduardogmisiuk) [inline lib/README.md:140]: This paragraph is kind of confusing. My suggestion is to create a list of places where the user should make sure that the id/name matches.

File: lib/samples/kelvin/README.md [hunk 2]
```diff
@@ -3,16 +3,30 @@
 This is a Dojot's flowbroker node that converts a Celcius temperature
```
Comment (eduardogmisiuk) [inline lib/samples/kelvin/README.md:3]: Celsius

File: lib/samples/kelvin/README.md [hunk 3]
```diff
@@ -22,20 +36,20 @@ JWT=$(curl -s -X POST http://localhost:8000/auth \
 -d '{"username": "admin", "passwd" : "admin"}' | jq -r ".jwt")
 ```
 
-Note: the previous command requires the `jq` command, you can install it on ubuntu
-with the following command:
+Note: the previous command requires the `jq`and `curl` command, you can install it on on Debian-based Linux distributions with the following command:
```
Comment (eduardogmisiuk) [inline lib/samples/kelvin/README.md:39]: \`jq\` and

File: lib/samples/kelvin/README.md [hunk 4]
```diff
@@ -22,20 +36,20 @@ JWT=$(curl -s -X POST http://localhost:8000/auth \
...
 ```
 sudo apt-get install jq
+sudo apt-get install curl
```
Comment (eduardogmisiuk) [inline lib/samples/kelvin/README.md:42]: Merge in one command.

File: lib/samples/kelvin/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "dojot-sample-node",
-  "version": "1.0.0",
+  "name": "dojot-kevin-node-package-sample",
+  "version": "0.1.0",
   "description": "This is a sample of node that transforms a given temperature from celcius to kenvin",
```
Comment (eduardogmisiuk) [inline lib/samples/kelvin/package.json:4]: from Celsius to Kelvin

**Review by eduardogmisiuk — CHANGES_REQUESTED** [review]

File: lib/README.md [hunk 6]
```diff
@@ -41,25 +49,25 @@ A example of content in a  __Message Catalog__:
...
 With namespace, the namespace will be the __id__ of node
-```javascript 
+```javascript
 console.log(RED._("__id__:myNode.message1"));
 ```
 
-##### Editor 
+##### Editor
 
 Any HTML element provided in the node template can specify a data-i18n attribute to provide the message identify to use. For example:
```
[the remark this answers is not in the document]
Comment (mprevide) [reply lib/README.md:68]: It must be 'identity'. I'll try to remake that phrase.

**Review by thiagodpf — APPROVED** [review]
it looks great to me!

File: lib/README.md [hunk 7]
```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
 ```
 Now the node will be available on the FlowBroker Dojot's interface.
 
-#### Making changes to the source code of the node and seeing them reflected
+#### Live Reloading your node
 
-**Note:** If you need to the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
+**Note:** If you need to change the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
```
Comment (eduardogmisiuk) [inline lib/README.md:131]: and see them reflected

File: lib/README.md [hunk 8]
```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
...
 
 ```sh
 curl -X DELETE -H "Authorization: Bearer ${JWT}" http://localhost:8000//flows/v1/node/<node name> -H 'content-type: application/json'
 ```
 
-**ATTENTION**: The `id` to add the node via API (when request `/flows/v1/node`) must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`. And within the html called in the `getNodeRepresentationPath` method also in the class that extends `dojot.DataHandlerBase`, the references inside the html `data-template-name=`, `data-help-name=`, `registerType(..` ,  and `RED._("...` must have this same `id`/`name`.
+#### **ATTENTION**: Avoiding problems with `id` and `name`
 
-This lib has been developed and tested using node v8.14.x
+To facilitate adding the node to the flowbroker, try to follow these intrusions:
```
Comment (eduardogmisiuk) [inline lib/README.md:141]: Suggestion:

To facilitate the addition of a node to the Flowbroker, follow these instructions:

File: lib/README.md [hunk 9]
```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
...
+
+##### **When adding the node via API**
+
+The `id` when request `/flows/v1/node` must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`.
```
Comment (eduardogmisiuk) [inline lib/README.md:145]: Suggestion:

The \`id\` sent in a request to the \`/flows/v1/node\` endpoint must be the same as the  \`name\` and the \`id\` defined in the \`getMetadata\` function inside the class that extends \`dojot.DataHandlerBase\`.

File: lib/README.md [hunk 10]
```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
...
+
+##### **When creating the node front end - html**
+
+When creating the html called in the `getNodeRepresentationPath` method also in the class that extends `dojot.DataHandlerBase` , the references within this html listed below must have the same `id`/`name`.
```
Comment (eduardogmisiuk) [inline lib/README.md:149]: html -> HTML
method also -> method and also

File: lib/README.md [hunk 11]
```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
...
+#### **ATTENTION**: Avoiding problems with `id` and `name`
 
-This lib has been developed and tested using node v8.14.x
+To facilitate adding the node to the flowbroker, try to follow these intrusions:
+
+##### **When adding the node via API**
+
+The `id` when request `/flows/v1/node` must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`.
+
+##### **When creating the node front end - html**
```
Comment (eduardogmisiuk) [inline lib/README.md:147]: HTML

File: README.md [hunk 12]
```diff
@@ -1,66 +1,175 @@
...
+
+Before proceeding, **make sure you configure your environment**.
+
+#### Flowbroker orchestrator
+
+##### General configurations
+
+Key                    | Purpose                       | Default Value      | Valid Values  |
+---------------------- | ----------------------------- | -------------------| --------------|
+AMQP_URL               | RabbitMQ host address    | amqp://rabbitmq  | url
```
Comment (eduardogmisiuk) [inline README.md:86]: These AMQP parameters are not alphabetically ordered.

**Review by eduardogmisiuk — APPROVED** [review]
Nice!

**Review by thiagodpf — APPROVED** [review]
:1st_place_medal:

**Pull request merged**